--- /etc/conf.d/docker.bak	2025-07-18 15:07:40.437456833 +0200
+++ /etc/conf.d/docker	2025-07-18 15:07:51.181146140 +0200
@@ -17,7 +17,7 @@
 #DOCKER_PIDFILE="/run/docker.pid"
 
 # Settings for process limits (ulimit)
-#DOCKER_ULIMIT="-c unlimited -n 524288 -u unlimited"
+DOCKER_ULIMIT="-c unlimited -n 524288"
 
 # seconds to wait for sending SIGTERM and SIGKILL signals when stopping docker
 #DOCKER_RETRY="TERM/60/KILL/10"
